project/frontend/webpack.config.js

```diff
@@ -33,8 +33,16 @@ module.exports = {
             presets: [
               "@babel/preset-env",
               "@babel/react",
+              "@babel/preset-react",
               {
-                plugins: ["@babel/plugin-proposal-class-properties"],
+                plugins: [
+                  [
+                    "@babel/plugin-proposal-class-properties",
+                    {
+                      loose: true,
+                    },
+                  ],
+                ],
```

Comment:
Is this actually how you're supposed to format this stuff? 😬

Reply:
Looks weird but it is yeah, or at least the formatter says so